en/docs/deploy/overview.md

```
@@ -0,0 +1,60 @@
+---
+title: Deploy Overview
+description: Learn about the different deployment options for your WSO2 Integrator projects and how to choose the right one for your needs.
+keywords: [wso2 integrator, deployment, wso2 cloud, docker, kubernetes, vm-based deployment, integration control plane]
+---
+{/* TODO: Work in progress */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)